kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/bluetooth/AdvertisingReportRecord.java

```diff
@@ -0,0 +1,53 @@
+package org.eclipse.kura.bluetooth;
```

Comment:
1. Missing license header
2. Missing Javadoc
3. Add ``@since`` Javadoc tag like done [here](https://github.com/eclipse/kura/blob/develop/kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/data/transport/listener/DataTransportListener.java)